docs-client/package.json

```diff
@@ -15,6 +15,7 @@
     "jsonminify": "^0.4.1",
     "react": "^16.6.0",
     "react-dom": "^16.6.0",
+    "react-dropdown": "^1.6.2",
```

Comment:
I guess you can use `<Select/>` provided by Material UI, which is our current UI toolkit? https://material-ui.com/demos/selects/ (I wish to migrate to something else like Blueprint later though..)

Reply:
Had a chat with @minwoox offline and he told me that he chose `react-dropdown` intentionally because it was easier to work with than `Select`. Given that we will move away from Material UI, I think it's fine as it is.